core/src/main/java/com/linecorp/armeria/server/cors/CorsPolicy.java

```diff
@@ -302,12 +302,32 @@ void setCorsPreflightResponseHeaders(ResponseHeadersBuilder headers) {
     }
 
     void setCorsAllowCredentials(ResponseHeadersBuilder headers) {
-        // The string "*" cannot be used for a resource that supports credentials.
-        // https://www.w3.org/TR/cors/#resource-requests
-        if (credentialsAllowed &&
-            !ANY_ORIGIN.equals(headers.get(HttpHeaderNames.ACCESS_CONTROL_ALLOW_ORIGIN))) {
-            headers.set(HttpHeaderNames.ACCESS_CONTROL_ALLOW_CREDENTIALS, "true");
-        }
+        CorsHeaderUtil.setCorsAllowCredentials(headers, this);
+    }
+
+    public Set<AsciiString> getExposedHeaders() {
+        return exposedHeaders;
+    }
+
+    public boolean isAllowAllRequestHeaders() {
+        return allowAllRequestHeaders;
+    }
+
+    public Set<AsciiString> getAllowedRequestHeaders() {
```

Comment:
We don't need this method because `allowedRequestHeaders()` does the exactly same job.